src/sentry/eventstream/kafka/protocol.py

```diff
@@ -74,7 +74,7 @@ class InvalidVersion(Exception):
     pass
 
 
-def get_task_kwargs_for_message(value):
+def get_task_kwargs_for_message(value) -> Optional[Mapping[str, Any]]:
```

Comment:
Should it be a TypedDict?

Reply:
I am planning to refactor code around this in another PR. Hence want to keep it simple for now